src/tests/topp/THIRDPARTY/CometAdapter_1_out.idXML

```diff
@@ -1,138 +1,171 @@
+				<UserParam type="string" name="CometAdapter:1:decoy_string_position" value="prefix"/>
+				<UserParam type="string" name="CometAdapter:1:missing_decoy_action" value="warn"/>
+				<UserParam type="string" name="CometAdapter:1:write_protein_sequence" value="false"/>
+				<UserParam type="string" name="CometAdapter:1:write_protein_description" value="false"/>
+				<UserParam type="string" name="CometAdapter:1:keep_unreferenced_proteins" value="false"/>
+				<UserParam type="string" name="CometAdapter:1:unmatched_action" value="error"/>
+				<UserParam type="int" name="CometAdapter:1:aaa_max" value="3"/>
+				<UserParam type="int" name="CometAdapter:1:mismatches_max" value="0"/>
+				<UserParam type="string" name="CometAdapter:1:IL_equivalent" value="false"/>
+				<UserParam type="string" name="CometAdapter:1:allow_nterm_protein_cleavage" value="true"/>
+				<UserParam type="string" name="CometAdapter:1:name" value="auto"/>
+				<UserParam type="string" name="CometAdapter:1:specificity" value="auto"/>
+				<UserParam type="string" name="extra_features" value="isotope_error,COMET:deltaCn,COMET:deltaLCn,COMET:lnExpect,MS:1002252,MS:1002255,COMET:lnNumSP,COMET:lnRankSP,COMET:IonFrac"/>
+				<UserParam name="EnzymeTermSpecificity" type="string" value="full" />
+	</SearchParameters>
+	<IdentificationRun date="2024-11-29T00:23:26" search_engine="Comet" search_engine_version="# comet_version 2024.02 rev. 0" search_parameters_ref="SP_0" >
+		<ProteinIdentification score_type="" higher_score_better="true" significance_threshold="0" >
+			<ProteinHit id="PH_0" accession="test2_rev" score="0.0" sequence="" >
+				<UserParam type="string" name="target_decoy" value="decoy"/>
+			</ProteinHit>
+			<ProteinHit id="PH_1" accession="BSA3" score="0.0" sequence="" >
+				<UserParam type="string" name="target_decoy" value="target"/>
+			</ProteinHit>
+			<UserParam type="stringList" name="spectra_data" value="[C:/OpenMS/src/tests/topp/THIRDPARTY/spectra_comet.mzML]"/>
+		</ProteinIdentification>
+		<PeptideIdentification score_type="expect" higher_score_better="false" significance_threshold="0.0" MZ="520.263366031899977" RT="2655.099999999999909" spectrum_reference="spectrum=0" >
+			<PeptideHit score="0.0104" sequence="DFASSGGYVLHLHR" charge="3" aa_before="[" aa_after="E" start="0" end="13" protein_refs="PH_0" >
+				<UserParam type="string" name="MS:1002258" value="28"/>
+				<UserParam type="string" name="MS:1002259" value="52"/>
+				<UserParam type="string" name="num_matched_peptides" value="1"/>
+				<UserParam type="int" name="isotope_error" value="0"/>
+				<UserParam type="float" name="MS:1002252" value="3.734"/>
+				<UserParam type="float" name="COMET:xcorr" value="3.734"/>
+				<UserParam type="float" name="MS:1002253" value="0.0"/>
+				<UserParam type="float" name="COMET:deltaCn" value="0.0"/>
+				<UserParam type="float" name="MS:1002255" value="1096.0"/>
+				<UserParam type="float" name="COMET:spscore" value="1096.0"/>

```

Comment:
these scores changed slightly.
Is this expected?

Reply:
From: https://github.com/UWPR/Comet/releases/tag/v2024.01.0
- Simplify the spectral processing for Sp scoring (preliminary score) by just taking the raw binned spectra and normalizing the max intensity to 100.
is what I found